src/main.cpp

```diff
@@ -2878,6 +2884,8 @@ bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockIndex* pin
                                      REJECT_INVALID, "bad-fwd-not-recorded");
                 }
             }
+
+            // a bwt request carries no events to schedule, nothing to be done for tx.GetVBwtRequestOut() 
```

Comment:
schedule sidechain events for MCBTR needed to update sc maturing balance

Reply:
Done in mbtr_introduction_final_fixes